R/get_stan_data.R

```diff
@@ -20,6 +20,8 @@
 #' @param params a dataframe of parameter names and numeric values
 #' @param compute_likelihood indicator variable telling stan whether or not to
 #' compute the likelihood, default = `1`
+#' @param dist_matrix Distance matrix for spatial correlation in distance
+#' correlation function.
```

Comment:
Confirming this is not a dataframe but in fact a numeric matrix? Perhaps specify that it is  n_sites x n_sites